packages/jest-changed-files/src/git.js

```diff
@@ -4,14 +4,16 @@
  * This source code is licensed under the BSD-style license found in the
  * LICENSE file in the root directory of this source tree. An additional grant
  * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
  */
 
 'use strict';
 
 const path = require('path');
 const childProcess = require('child_process');
 
-function findChangedFiles(cwd) {
+function findChangedFiles(cwd: string): Promise {
```

Comment:
can we make this a little bit more specific? `Promise<Array<Path>>` would be nice I think!